MainDemo.Wpf/PopupBox.xaml

```diff
@@ -0,0 +1,198 @@
+﻿<UserControl x:Class="MaterialDesignDemo.PopupBox"
+             xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+             xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+             xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006" 
+             xmlns:d="http://schemas.microsoft.com/expression/blend/2008" 
+             xmlns:local="clr-namespace:MaterialDesignDemo"
+             xmlns:materialDesign="http://materialdesigninxaml.net/winfx/xaml/themes"
+             xmlns:system="clr-namespace:System;assembly=System.Runtime"
+             mc:Ignorable="d" 
+             d:DesignHeight="450" d:DesignWidth="800">
+  <UserControl.Resources>
+    <ResourceDictionary>
+      <!-- This is needed to avoid runtime exceptions?! Seems like this might be a bug? -->
+      <ResourceDictionary.MergedDictionaries>
+        <ResourceDictionary Source="pack://application:,,,/MaterialDesignThemes.Wpf;component/Themes/MaterialDesignTheme.PopupBox.xaml" />
+      </ResourceDictionary.MergedDictionaries>
```

Comment:
I was quite surprised that I had to explicitly include the `MaterialDesignTheme.PopupBox.xml` for this to work. If left out, it fails to find the (default) Style for the `PopupBox`. I would assume this was already loaded in `App.Xaml` and therefore should not be needed here?

Reply:
Yeah perhaps it should then. It feels a little weird to have to include that explicitly.

Reply:
Upon further inspection I think it is working as expected.
All controls have their default style loaded by the App.xaml only referencing the Defaults resource dictionary, with users needing to reference the other resource dictionaries, so for now I think the behavior is correct.